src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/StringLiteralEqualityCheck.java

```diff
@@ -84,6 +84,8 @@
  *
  * @since 3.2
  * @noinspection HtmlTagCanBeJavadocTag
+ * @noinspectionreason HtmlTagCanBeJavadocTag - encoded symbols were not decoded
+ *      when replaced with Javadoc tag
```

Comment:
This was the case that caused the need for https://github.com/checkstyle/checkstyle/pull/11880/files#r917429288